CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The `next` version of 4.30 is now available. Planned release date is June 2024.
 
 ### Components breaking changes
 
+- In `@arcgis/map-components@next`, for non-view based components the `arcgis<nameOfComponent><nameofEvent>` event naming pattern has been removed. Use the `arcgis<nameOfEvent>` naming pattern instead. For example, this change affects `arcgis-feature-table` and `arcgis-table-list`. It does not affect `arcgis-map` or `arcgis-scene`. 
```

Comment:
I'd use components other than `arcgis-feature-table` and `arcgis-table-list` as an example until they're publicly documented.

Reply:
Fixed.